### PR TITLE
Docs Walkthrough

### PR DESCRIPTION
**Pull request by mbkuhn**
## Summary

Walkthrough section of documentation. 3 main components:
1. Compiling using Spack, with Exawind-Manager
2. Precursor (ABL) walkthrough - this is 2 parts
3. Turbine simulation walkthrough - for openfast-coupled actuator disks

3 example input files

## Pull request type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: docs/sphinx/walkthrough/precursor_inp.txt [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+#.......................................#
+ICNS.source_terms                        = BoussinesqBuoyancy CoriolisForcing ABLForcing
+incflo.velocity                          = 10.0 0.0 0.0
+ABLForcing.abl_forcing_height            = 86.5
+CoriolisForcing.latitude                 = 36.607322      # Southern Great Planes
+CoriolisForcing.north_vector             = 0.0 1.0 0.0
+CoriolisForcing.east_vector              = 1.0 0.0 0.0
+BoussinesqBuoyancy.reference_temperature = 290.0
+ABL.reference_temperature                = 290.0
+ABL.temperature_heights                  = 0.0 600.0 700.0 1700.0    # Make sure the top height here goes above the domain height
```
Comment (mbkuhn) [inline docs/sphinx/walkthrough/precursor_inp.txt:57]: not sure about this comment

File: docs/sphinx/walkthrough/spinup_inp.txt [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+#.......................................#
+ICNS.source_terms                        = BoussinesqBuoyancy CoriolisForcing ABLForcing
+incflo.velocity                          = 10.0 0.0 0.0
+ABLForcing.abl_forcing_height            = 86.5
+CoriolisForcing.latitude                 = 36.607322                 # Southern Great Plains
+CoriolisForcing.north_vector             = 0.0 1.0 0.0
+CoriolisForcing.east_vector              = 1.0 0.0 0.0
+BoussinesqBuoyancy.reference_temperature = 290.0
+ABL.reference_temperature                = 290.0
+ABL.temperature_heights                  = 0.0 600.0 700.0 1700.0    # Make sure the top height here goes above the domain height
```
Comment (mbkuhn) [inline docs/sphinx/walkthrough/spinup_inp.txt:50]: same for this one

File: docs/sphinx/walkthrough/precursor_inp.txt [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+incflo.verbose                           =   0          # incflo_level
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            GEOMETRY & BCs             #
+#.......................................#
+geometry.prob_lo                         = 0.       0.     0.   # Lo corner coordinates
+geometry.prob_hi                         = 2560.  2560.  1280.  # Hi corner coordinates
+amr.n_cell                               = 128 128 64           # Grid cells at coarsest AMRlevel
+amr.max_level                            = 2                    # Max AMR level in hierarchy
```
Comment (mbkuhn) [inline docs/sphinx/walkthrough/precursor_inp.txt:38]: not sure about using refinement zones in precursor

**Comment by rybchuk** [comment]
Excited to see!!

**Comment by mbkuhn** [comment]
Matteo has gone through the walkthrough himself, confirming the instructions in each part work as intended -- this includes running the 3 linked simulations.

File: docs/sphinx/walkthrough/index.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Walkthrough
+===========
+
+Insert overview of walkthrough: notes on compiling, precursor simulation setup, and turbine simulation.
```
[the remark this answers is not in the document]
Comment (mbkuhn) [reply docs/sphinx/walkthrough/index.rst:4]: oh, good catch. I can replace that now!

**Review by marchdf — APPROVED** [review]

**Pull request merged**